.github/workflows/pr-automation.yml

```diff
@@ -115,3 +115,29 @@ jobs:
             
               console.log(result);
             }
+
+  # check PR assignee - if no one is assigned, assign to person who merge PR
+  assignees:
+    permissions:
+      issues: write
+      pull-requests: write
+
+    if: github.event.action == 'closed' && github.event.pull_request.merged && toJSON(github.event.pull_request.assignees) == '[]'
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Assign PR
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7
+        with:
+          script: |
+            
+            console.log(context.issue.assignees);
+                        
+            const result = await github.rest.issues.update({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              assignees: [ '${{ github.event.pull_request.merged_by.login }}' ]
```

Comment:
We could keep the current assignee as well, if that's okay for you.

Reply:
Will only execute if no one is assigned.
See line 125:
```
if: ... && toJSON(github.event.pull_request.assignees) == '[]'
```

Reply:
Then okay by me!